frontend/src/i18n/locales/en.json

```
@@ -0,0 +1,370 @@
+    "shopOurStory": "Shop Our Story",
+    "meetOurTeam": "Meet Our Team",
+    "ourMission": "Our Mission",
+    "missionDescription": "To make beautiful, sustainable, and functional design accessible to everyone. We believe that your home should be a reflection of your values—simple, authentic, and thoughtfully curated.",
+    "missionQuote": "“Design is not just what it looks like and feels like. Design is how it works. We create spaces that work beautifully for real life.”",
+    "missionAuthor": "- Sarah Chen, Founder",
+    "ourStory": "Our Story",
+    "ourValues": "Our Values",
+    "valuesDescription": "These principles guide everything we do, from product selection to customer service.",
+    "ourTeam": "Our Team",
+    "teamDescription": "The passionate people behind Minimal, dedicated to bringing you exceptional design and service.",
+    "testimonials": "Testimonials",
```

Biome (2.4.14)

[error] The key testimonials was already declared.

(lint/suspicious/noDuplicateObjectKeys)